bears/html/BootLintBear.py

```diff
@@ -1,34 +1,24 @@
-import re
-
-from coalib.bearlib.abstractions.Lint import Lint
-from coalib.bears.LocalBear import LocalBear
+from coalib.bearlib.abstractions.Linter import linter
 from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
 from coalib.settings.Setting import typed_list
 
 
-class BootLintBear(LocalBear, Lint):
-    executable = 'bootlint'
-    output_regex = re.compile(
-        r'(.+?):*(?P<line>\d*):+'
-        r'(?P<col>\d*)\s(?P<severity>.)\d+'
-        r'\s(?P<message>.+)')
-    severity_map = {
-        "W": RESULT_SEVERITY.NORMAL,
-        "E": RESULT_SEVERITY.MAJOR
-    }
-
-    def _get_groupdict(self, match):
-        groups = Lint._get_groupdict(self, match)
-        if groups["line"] == "" and groups["col"] == "":
-            del groups["line"], groups["col"]
-        return groups
-
-    def run(self, filename, file, bootlint_ignore: typed_list(str)=[]):
-        '''
-        Checks the code with ``bootlint`` on each file separately.
+@linter(executable='bootlint',
+        output_format='regex',
+        output_regex=r'.+:(?P<line>\d*):(?P<column>\d*) (?P<severity>.)\d+ '
+                     r'(?P<message>.+)',
```

Comment:
regex looks slightly different after the line and before it, but i'm sure you tested it and it is parsed like this too :) this one looks a little more efficient